### DISCUSSION TITLE
MY APPLICATION IS REJECTED THOUGH I HAVE A VALID SCHOOL ISSUED EMAIL ADDRESS AND IDENTITY CARD

### DISCUSSION THREAD

**Discussion by fykaa** [community/community#27664] [Discussions]
Please respond to this issue or provide me a GitHub mail to write this issue and request a thorough review of my application.

**Comment by Gisgar3** [comment]
You can use [this](https://support.github.com/contact/education) to contact GitHub directly regarding issues with GitHub Education. Hope this helps!
